src/app/cli/src/coda.ml

```diff
@@ -509,13 +519,23 @@ let setup_daemon logger =
         | Some v ->
             Some v
         | None ->
-            let%bind daemon_config = daemon_config in
-            let%bind json_val =
-              to_option Fn.id (member keyname daemon_config)
+            (* Load value from the first config file that both
```

Comment:
since the search is on the reversed list, is it the latest config file, rather than the first config file?